frontend/redux/selectors.js

```diff
@@ -24,3 +24,22 @@ export const isLoading = state => Object
 
 export const demographics = state => state.demographics.questions;
 export const demographicsOrder = state => state.demographics.order;
+
+export const selectWorkload = function(state) {
+  const images = [];
+  const answers = store.getState().demographics.answers;
+
+  Object.keys(answers).map((i) => {
+    images.push({
+      id: parseInt(i),
+      demographics: answers[i],
+    });
+  });
```

Comment:
Could be simplified to
```js
const images = Object.keys(answers).map((i) => ({
  id: parseInt(i, 10),
  demographics: answers[i],
});
```
adding the radix parameter to the parseInt will also resolve one of the lint errors